test/test-applications/integrations/Samples.ServiceStack.Redis/Samples.ServiceStack.Redis.csproj

```diff
@@ -6,6 +6,8 @@
     <!-- Required to build multiple projects with the same Configuration|Platform -->
     <GenerateAssemblyInfo>false</GenerateAssemblyInfo>
     <GenerateTargetFrameworkAttribute>false</GenerateTargetFrameworkAttribute>
+    <!-- Ignore Found conflicts between different versions of "System.ValueTuple" (and others) that could not be resolved.   -->
```

Comment:
Should we try to fix these conflicts instead of ignoring the warnings?

Reply:
That would be great, but I'm not sure exactly why they're there. I _guess_ it's related to the .NET STandard 1.5 target  in StackExchange.Redis 1.2.6, but I'm not sure how to fix that? 🤷

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/1524/commits/95099cfe6984ee156419b8f2721be3327aa8b498